pkg/console/network.go

```diff
@@ -133,3 +134,15 @@ func listNetworkHardware() (map[string]networkHardwareInfo, error) {
 
 	return m, nil
 }
+
+func setHostname(hostname string) error {
+	// NOTE: Can't use Golang's syscall.Sethostname
+	// because it sets the "trasient hostname", not "static hostname".
+	// wicked uses static hostname to make the DHCP request.
+	cmd := exec.Command("hostnamectl", "set-hostname", fmt.Sprintf("%q", hostname))
```

Comment:
LGTM.

FWIW. yip also sets hosts file.
https://github.com/mudler/yip/blob/master/pkg/plugins/hostname.go#L50-L59

Reply:
This seems like a neater solution to set the hostname! I'll send another PR for this.